skimage/segmentation/slic_superpixels.py

```diff
@@ -139,11 +139,17 @@ def slic(image, n_segments=100, compactness=10., max_iter=10, sigma=0,
         Run SLIC-zero, the zero-parameter mode of SLIC. [2]_
     start_label: int, optional
         The labels' index start. Should be 0 or 1.
+
+        .. versionadded:: 0.17
+           ``start_label`` is introduced in 0.17
     mask : 2D ndarray, optional
         If provided, superpixels are computed only where mask is True,
         and seed points are homogeneously distributed over the mask
         using a K-means clustering strategy.
 
+        .. versionadded:: 0.17
+           ``mask`` is introduced in 0.17
```

Comment:
Change to "was"? same above.

Reply:
I forgot the syntax so I copied this from the `filters.median` docstring which used "is". I agree that "was" sounds better